distribution/docs/src/main/resources/_contents/_data/data-intro-contents.adoc

```diff
@@ -53,138 +53,164 @@ The default metacard type has the following required attributes.
 Though the following attributes are required on all metacard types, setting their values is optional except for ID.
```

Comment:
Now that we have attribute injection, and so many metacards that are not `resource` metacards, does this section even make sense anymore? We should certainly *recommend* the core set at a minimum, but anything that is strictly required should just be injected where not present. Or should it? If we don't inject then we have to fail-fast or allow for any attribute to be missing in all our implementations.